sudospawner/spawner.py

```diff
@@ -59,6 +63,9 @@ def do(self, action, **kwargs):
         cmd.append(self.sudospawner_path)
         if self.debug_mediator:
             cmd.append('--logging=debug')
```

Comment:
This will set it twice. Instead of `cmd.append` here, set self.mediator_log_level = 'debug'

Reply:
It wouldn't set it twice because of the `elif`, but I have updated it to set `self.mediator_log_level` with priority still on `debug_mediator`.